@navikt/internal/css/timeline.css

```
@@ -0,0 +1,222 @@
+.navdsi-timeline {
+  position: relative;
+  display: grid;
+  grid-template-columns: auto minmax(0, 1fr);
+  width: 100%;
+}
+
+.navdsi-timeline__axislabels {
+  position: relative;
+  height: 1rem;
+  box-sizing: content-box;
+  margin-bottom: 16px;
+  grid-column: 2;
+  margin-left: 2.375rem;
+}
+
+.navdsi-timeline__row__label {
+  white-space: nowrap;
+  width: auto;
+  margin-right: 1.125rem;
+  grid-column: 1;
+  font-size: var(--navds-font-size-medium);
+  height: 1.5rem;
+  display: flex;
+  align-items: center;
+  gap: var(--navds-spacing-4);
+}
+
+.navdsi-timeline__axislabels > * {
+  position: absolute;
+  font-size: var(--navds-font-size-small);
+  color: var(--navds-global-color-gray-600);
+}
+
+.navdsi-timeline__row {
+  display: flex;
+  background: var(--navds-global-color-gray-50);
+  margin: 0 0 var(--navds-spacing-5);
+  padding-left: 2.375rem;
+  grid-column: 2;
+}
+
+.navdsi-timeline__row:last-of-type {
+  margin-bottom: 0;
+}
+
+.navdsi-timeline__row__periods {
+  height: var(--navds-font-line-height-large);
+  position: relative;
+  width: 100%;
+  margin: 0;
+}
+
+.navdsi-timeline__row__periods li {
+  all: unset;
+}
+
+.navdsi-timeline__period {
+  height: calc(100% - 2px);
+  border-radius: 20px;
+  position: absolute;
+  border: 1px solid;
+  padding: 0;
+  display: flex;
+  align-items: center;
+  z-index: 2;
+}
+
+.navdsi-timeline__period > * {
+  padding-left: var(--navds-spacing-2);
+}
+
+.navdsi-timeline__period--success {
+  background: var(--navds-global-color-green-50);
+  border-color: var(--navds-global-color-green-500);
+}
+
+.navdsi-timeline__period--success[data-clickable]:hover {
+  background: var(--navds-global-color-green-100);
+}
+
+.navdsi-timeline__period--warning {
+  background: var(--navds-global-color-orange-50);
+  border-color: var(--navds-global-color-orange-600);
+}
+
+.navdsi-timeline__period--warning[data-clickable]:hover {
+  background: var(--navds-global-color-orange-100);
+}
+
+.navdsi-timeline__period--danger {
+  background: var(--navds-global-color-red-50);
+  border-color: var(--navds-global-color-red-500);
+}
+
+.navdsi-timeline__period--danger[data-clickable]:hover {
+  background: var(--navds-global-color-red-100);
+}
+
+.navdsi-timeline__period--information {
+  background: var(--navds-global-color-lightblue-50);
+  border-color: var(--navds-global-color-lightblue-700);
+}
+
+.navdsi-timeline__period--information[data-clickable]:hover {
+  background: var(--navds-global-color-lightblue-100);
+}
+
+.navdsi-timeline__period--default {
```

Foreslår at default erstattes med `neutral` på tvers, men at `neutral` er default valgt. 🤔 
```suggestion
.navdsi-timeline__period--neutral {
```